usr/src/cmd/cmd-inet/usr.lib/mdnsd/DNSDigest.c

```diff
@@ -1,6 +1,7 @@
 /* -*- Mode: C; tab-width: 4 -*-
  *
  * Copyright (c) 2002-2011 Apple Inc. All rights reserved.
+ * Copyright (c) 2016 by Delphix. All rights reserved.
```

Comment:
It appears there are no changes to this file other than a copyright addition. Was that intentional?